src/include/shared/sgxlkl_enclave_config_gen.h

```diff
@@ -0,0 +1,148 @@
+#ifndef SGXLKL_ENCLAVE_CONFIG_GEN_H
+#define SGXLKL_ENCLAVE_CONFIG_GEN_H
+
+/* Automatically generated from ../../tools/schemas/enclave-config.schema.json; do not modify. */
```

Comment:
If this is auto-generated, why are we committing it to the repo?

Reply:
Yes. This will change very infrequently, so there's no point in re-generating them in every build. Checking them in avoids people having to install dependencies (like Python and a number of pip packages). The dependencies are set up, so if someone changes something, it will try to regenerate them (and fail if they don't have the dependencies installed).

Reply:
I had a quick look at the Python generator and it only uses libraries of the standard library for generation. The validation part that's still in there isn't used for generation and should go anyway (as it's done by `sgx-lkl-cfg validate`). So the only dependency is to have python 3, which we need anyway.

Reply:
Great, thanks for checking that @letmaik . In that case I'm fine with re-generating.

Reply:
Done